servant-mock/src/Servant/Mock.hs

```diff
@@ -160,6 +160,9 @@ instance HasMock Raw where
 
     where genBody = pack <$> generate (vector 100 :: Gen [Char])
 
+instance HasMock rest => HasMock (WithNamedConfig name config rest) where
```

Comment:
Doesn't there need to be an instance for `(WithNamedConfig name config s) :> sublayout` too?